api.md

```diff
@@ -317,6 +335,7 @@ See [boostrapping description][multinode-bootstrap] for details.
 | `port`               | Port to use on the remote data node. The default is the PostgreSQL port used by the access node on which the function is executed. |
 | `if_not_exists`      | Do not fail if the data node already exists. The default is `FALSE`. |
 | `bootstrap`          | Bootstrap the remote data node. The default is `TRUE`. |
+| `password`           | Password for authenticating with the remote data node during bootstrapping or validation. A password only needs to be provided if the data node requires password authentication and a password for the user does not exist in a local password file on the access node. |
```

Comment:
What happens if a password is provided by it does not use password authentication?

Reply:
Nothing, AFAIK. The method of authentication is determined by the data node.

Reply:
There is a difference between giving an error if the authentication method does not require a password and just ignoring the password if the authentication method does not require the password. If it is just ignored in this case, it is good to mention it here.

Reply:
I don't see why this is necessary and we generally don't list all error cases for optional parameters. The explanation here is already long.